R/AddCardModule.R

```diff
@@ -47,9 +47,6 @@ add_card_button_ui <- function(id) {
   # Buttons with custom css and
   # js code to disable the add card button when clicked to prevent multi-clicks
   shiny::tagList(
-    shiny::singleton(
-      shiny::tags$head(shiny::includeCSS(system.file("css/custom.css", package = "teal.reporter")))
```

Comment:
@gogonzo any reason why this is repeated throughout the `teal_reporter`?

Once the previewer's tab is added the CSS should be attached to DOM